### ISSUE TITLE
question about learning rate

### ISSUE THREAD

**Issue by liu-nlper** [brightmart/bert_language_understanding#11]
Hi, what is the learning rate when you train TextCNN(No-pretrain)?

MIDDLE SIZE DATASET(<a href='https://pan.baidu.com/s/1HUzBXB_-zzqv-abWZ74w2Q'>cail2018</a>, 450k)

Model                        | TextCNN(No-pretrain)| TextCNN(Pretrain-Finetuning)| Gain from pre-train 
---                          | ---                 | ---                         | -----------     
F1 Score after 1 epoch       |  0.09               | 0.58                        |  0.49        
F1 Score after 5 epoch       |  0.40               | 0.74                        |  0.35    
F1 Score after 7 epoch       |  0.44               | 0.75                        |  0.31                         
F1 Score after 35 epoch      |  0.58               | 0.75                        |  0.27                                                
Training Loss at beginning   |  284.0              | 84.3                        |  199.7             
Validation Loss after 1 epoch|  13.3               | 1.9                         |  11.4                 
Validation Loss after 5 epoch|  6.7                | 1.3                         |  5.4   
Training time(single gpu)    |  8h                 | 2h                          |  6h
----------------------------------------------------------------------------------------------

**Comment by brightmart** [comment]
the default learning rate as can find in tain_xxx.py

**Comment by liu-nlper** [comment]
I think leanring rate 0.00001 is too samll for Adam optimizer, the model may converge very slow, have you tried set learning rate as 0.001?

**Comment by liu-nlper** [comment]
I think leanring rate 0.00001 is too samll for Adam optimizer, the model may converge very slow without pre-training, have you tried set learning rate as 0.001?
